Gems/ROS2/Code/Source/RobotImporter/Utils/RobotImporterUtils.cpp

```diff
@@ -341,161 +342,224 @@ namespace ROS2::Utils
+            if (fileExistsCB(unresolvedPath))
             {
-                AZStd::string_view strippedUriPath = uriPath.Native().substr(FileSchemePrefix.size());
-                return AZ::IO::Path(strippedUriPath);
+                AZ_Trace("ResolveAssetPath", R"(Resolved Absolute Path: "%.*s")" "\n", 
+                    AZ_PATH_ARG(unresolvedPath));
+                return unresolvedPath;
             }
-
-            return AZStd::nullopt;
-        };
-
-        // Step 1: Attempt to resolved URI scheme paths
-        // libsdformat seems to convert package:// references to model:// references
-        // So the model:// URI prefix resolver is run first
-        const auto uriPrefixes = AZStd::to_array<UriPrefix>({
-            AZStd::move(modelUriPrefix),
-            AZStd::move(fileUriPrefix),
-            AZStd::move(packageUriPrefix)});
-        for (const UriPrefix& uriPrefix : uriPrefixes)
-        {
-            if (auto resolvedPath = uriPrefix.m_schemeResolver(unresolvedPath);
-                resolvedPath.has_value())
+            else
             {
-                AZ_Printf("ResolveURDFPath", R"(Resolved Path using URI Prefix "%.*s" -> "%.*s")" "\n", AZ_PATH_ARG(unresolvedPath),
-                    AZ_PATH_ARG(resolvedPath.value()));
-                return resolvedPath.value();
+                AZ_Trace("ResolveAssetPath", R"(Failed to resolve Absolute Path: "%.*s")" "\n", 
+                    AZ_PATH_ARG(unresolvedPath));
+                return "";
             }
         }
 
-        // At this point, the path has no URI scheme
-        if (unresolvedPath.IsAbsolute())
+        // The path is a relative path, so use the directory containing the base URDF/SDF file as the root path,
+        // and if the file can be found successfully, return the path. Otherwise, return an empty path as an error.
+        const AZ::IO::Path relativePath = AZ::IO::Path(baseFilePath.ParentPath()) / unresolvedPath;
+
+        if (fileExistsCB(relativePath))
         {
-            AZ_Printf("ResolveURDFPath", "Input Path is an absolute local filesystem path to : %s\n", unresolvedPath.c_str());
-            return unresolvedPath;
+            AZ_Trace("ResolveAssetPath", R"(Resolved Relative Path: "%.*s" -> "%.*s")" "\n", 
+                AZ_PATH_ARG(unresolvedPath), AZ_PATH_ARG(relativePath));
+            return relativePath;
         }
 
-        // The path is relative path, so append to the directory containing the .urdf file
-        const AZ::IO::Path resolvedPath = AZ::IO::Path(urdfFilePath.ParentPath()) / unresolvedPath;
-        AZ_Printf("ResolveURDFPath", "Input Path %s is being returned as is\n", unresolvedPath.c_str());
-        return resolvedPath;
+        AZ_Trace("ResolveAssetPath", R"(Failed to resolve Relative Path: "%.*s" -> "%.*s")" "\n", 
+            AZ_PATH_ARG(unresolvedPath), AZ_PATH_ARG(relativePath));
+        return "";
```

Comment:
repeat: returning using list initialization causes the default constructor to be invoked which is infinitesimally quicker. 
```suggestion
        return {};
```